website/docs/r/securityhub_insight.html.markdown

```diff
@@ -0,0 +1,388 @@
+* `start` - (Optional) A start date for the date filter. Required with `end` if `date_range` is not specified.
+
+### date_range Argument reference
+
+The `date_range` configuration block supports the following arguments:
+
+* `unit` - (Required) A date range unit for the date filter. Valid values: `DAYS`.
+* `value` - (Required) A date range value for the date filter, provided as an Integer.
+
+### Ip Filter Argument Reference
+
+The Ip filter configuration block supports the following arguments:
+
+* `cidr` - (Required) A finding's CIDR value.
+
+### Keyword Filter Argument Reference
+
+The keyword filter configuration block supports the following arguments:
+
+* `value` - (Required) A value for the keyword.
+
+### Map Filter Argument reference
+
+The map filter configuration block supports the following arguments:
+
+* `comparison` - (Required) The condition to apply to a string value when querying for findings. Valid values include: `EQUALS` and `NOT_EQUALS`.
+* `key` - (Required) The key of the map filter. For example, for `ResourceTags`, `Key` identifies the name of the tag. For `UserDefinedFields`, `Key` is the name of the field.
+* `value` - (Required) The value for the key in the map filter. Filter values are case sensitive. For example, one of the values for a tag called `Department` might be `Security`. If you provide `security` as the filter value, then there is no match.
+
+### Number Filter Argument reference
+
+The number filter configuration block supports the following arguments:
+
+~> **NOTE:** Only one of `eg`, `gte`, or `lte` must be specified.
+
+* `eq` - (Optional) The equal-to condition to be applied to a single field when querying for findings, provided as a String.
+* `gte` - (Optional) The greater-than-equal condition to be applied to a single field when querying for findings, provided as a String.
+* `lte` - (Optional) The less-than-equal condition to be applied to a single field when querying for findings, provided as a String.
+
+### String Filter Argument reference
+
+The string filter configuration block supports the following arguments:
+
+* `comparison` - (Required) The condition to apply to a string value when querying for findings. Valid values include: `EQUALS`, `PREFIX`, `NOT_EQUALS`, `PREFIX_NOT_EQUALS`.
+* `value` - (Required) The string filter value. Filter values are case sensitive.
+
+### Workflow Status Filter Argument reference
+
+The workflow status filter configuration block supports the following arguments:
+
+* `comparison` - (Required) The condition to apply to a string value when querying for findings. Valid values include: `EQUALS`, `PREFIX`, `NOT_EQUALS`, `PREFIX_NOT_EQUALS`.
+* `value` - (Required) The string filter value. Valid values include: `NEW`, `NOTIFIED`, `SUPPRESSED`, and `RESOLVED`.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The Amazon Resource Name (ARN) of the insight.
```

Comment:
```suggestion
* `id` - ARN of the insight.
```

Personally, I think we should get rid of `Amazon Resource Name` anywhere in the docs. If you're using Terraform, you'll know.

Reply:
true!